config/filesystems.php

```diff
@@ -57,10 +57,10 @@
 
         's3' => [
             'driver' => 's3',
-            'key' => env('S3_KEY'),
-            'secret' => env('S3_SECRET'),
-            'region' => 'us-east-1',
-            'bucket' => env('S3_BUCKET', 'ds-rogue-qa'),
+            'key' => env('AWS_ACCESS_KEY'),
+            'secret' => env('AWS_SECRET_KEY'),
+            'region' => env('AWS_REGION', 'us-east-1'),
```

Comment:
Note that we don't currently set this from Terraform, so it'll fall back to the `us-east-1` (same as before).